spec/spec_helper.rb

```diff
@@ -12,6 +12,7 @@
 )
 SimpleCov.start "rails" do
   add_filter '/spec'
+  add_filter 'app/controllers/geoserver_controller.rb'
```

Comment:
Why is this left out? Hard to test? Can you add a comment?

Reply:
Yes, hard to test for. James added this code as a replacement reverse proxy for GeoServer awhile back.

Reply:
https://github.com/pulibrary/pulmap/pull/798